[TUTORIAL] Doc content review (#3092)
scaleway/docs-content: tutorials/sinatra/index.mdx

The easiest way to install Ruby and RubyGems is using the Ruby Version Manager (RVM). RVM allows you to have multiple Rubies, with their own sets of gems (gemsets), installed.

> 1. (Optional) Install `nodejs`, `gunpg2`, `ruby` and `curl`. When using Rails, is is required to install Node.js. The asset pipeline compiler of Rails requires a Javascript runtime. The version of `Node.js` that you install does not matter. If you do not use Rails, skip this and proceed to the next step.

Reviewer: ```suggestion
1. (Optional) Install `nodejs`, `gunpg2`, `ruby`, and `curl`. When using Rails, you must install Node.js, in any version you want. The asset pipeline compiler of Rails requires a JavaScript runtime. If you do not use Rails, skip this and proceed to the next step.
```
